src/main/java/com/spikes2212/command/genericsubsystem/commands/smartmotorcontrollersubsystem/MoveSmartMotorControllerSubsystemTrapezically.java

```diff
@@ -0,0 +1,52 @@
+package com.spikes2212.command.genericsubsystem.commands.smartmotorcontrollersubsystem;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.revrobotics.CANSparkMax;
+import com.spikes2212.command.genericsubsystem.smartmotorcontrollersubsystem.SmartMotorControllerSubsystem;
+import com.spikes2212.control.FeedForwardSettings;
+import com.spikes2212.control.PIDSettings;
+import com.spikes2212.control.TrapezoidProfileSettings;
+
+import java.util.function.Supplier;
+
+
+/**
+ * A command that moves a {@link SmartMotorControllerSubsystem} by running a trapezoid profile on its
+ * master's control loops.
+ *
+ * @author Yoel Perman Brilliant
+ * @see SmartMotorControllerSubsystem
+ * @see MoveSmartMotorControllerSubsystem
+ */
+public class MoveSmartMotorControllerSubsystemTrapezically extends MoveSmartMotorControllerSubsystem {
+
+    protected final TrapezoidProfileSettings trapezoidProfileSettings;
+
+    /**
+     * Constructs a new instance of {@link MoveSmartMotorControllerSubsystem}.
```

Comment:
Constructs a new instance of `{@link MoveSmartMotorControllerSubsystemTrapezically}`.

also add to params the `trapezoidProfileSettings`